user_docs/en/userGuide.t2t

```diff
@@ -2589,33 +2589,67 @@ To toggle the braille viewer from anywhere, please assign a custom gesture using
 The NVDA Python console, found under Tools in the NVDA menu, is a development tool which is useful for debugging, general inspection of NVDA internals or inspection of the accessibility hierarchy of an application.
 For more information, please see the [NVDA Developer Guide https://www.nvaccess.org/files/nvda/documentation/developerGuide.html].
 
-++ Add-ons Manager ++[AddonsManager]
-The Add-ons Manager, accessed by selecting Manage add-ons under Tools in the NVDA menu, allows you to install, uninstall, enable and disable add-on packages for NVDA.
+++ Add-on Store ++[AddonsManager]
+The Add-on Store, accessed by the Tools submenu in the NVDA menu, allows you to browse and manage add-on packages for NVDA.
 These packages are provided by the community and contain custom code that may add or change features in NVDA or even provide support for extra Braille displays or speech synthesizers.
 
-The Add-ons Manager contains a list that displays all the add-ons currently installed in your NVDA user configuration. 
-Package name, status, version and author are shown for each add-on, though further information such as a description and URL can be viewed by selecting the add-on and pressing the About add-on button.
-If there is help available for the selected add-on, you can access it by pressing the Add-on help button.
+When exiting the Add-on Store, you may need to restart NVDA so that your changes can take effect.
+NVDA will ask you to restart when exiting the Add-on Store if you have updated, installed, removed or changed the status of an add-on.
 
-To browse and download available add-ons online, press the Get add-ons button.
-This button opens the [NVDA Add-ons page https://addons.nvda-project.org/].
-If NVDA is installed and running on your system, you can open the add-on directly from the browser to begin the installation process as described below.
-Otherwise, save the add-on package and follow the instructions below.
+To access the Add-on Store from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
-To install an Add-on you previously obtained, press the Install button.
-This will allow you to browse for an add-on package (.nvda-addon file) somewhere on your computer or on a network.
++++ Browsing add-ons +++[AddonStoreBrowsing]
+The Add-on Store shows a list of add-ons in a panel.

```

Comment:
Possibly remove the mention of panels. Blind people are not really made aware of these panels in the hierarchy so could be confusing.

Reply:
My suggestion is:

Change: "in a panel" to "down the left half of the dialog".  It confirms where the list is in the dialog either for sighted users, or those showing the screen to sighted users.

Similarly, the next paragraph, starts: "Selecting an add-on will display the details for the add-on in the next panel.".

For the same reasons, change "in the next panel" to "on the right half of the dialog".